Client/Ecosia/AppDelegate/Leaves/TabManagerAppDelegate.swift

```diff
@@ -0,0 +1,32 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Storage
+
+final class TabManagerAppDelegate: AppDelegateLeaf {
+    
+    private var tabManager: TabManager?
+    private var profile: Profile
```

Comment:
Is `profile` ever used here? Can't it be removed?

Reply:
Can be removed yeah.